lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -1 +0,0 @@
-pipfiles/default/pinned-requirements.txt
```

Comment:
This seems a merge problem.

Reply:
Am I reading it correct? it deleted one line and added a new line with the same content as the original line?

Reply:
Not sure, but it looks a bit strange.

Reply:
For some odd reasons, this is actually replacing a symlink with a text file containing path to the originally linked file. I fixed it; and thanks @natefoo for identifying the issue.